plugins/modules/ipam_federation_federated_realm_info.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Infoblox Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: ipam_federation_federated_realm_info
+short_description: Retrieves information about existing Federated Realm
```

Comment:
```suggestion
short_description: Retrieves Federated Realms
```
For consistency purposes